src/components/Map.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import GoogleMapReact from 'google-map-react';
+
+function Map(){
+    const defaultLocation = {
+        center:{
+            lat: 59.95,
+            lng: 30.33
+        },
```

Comment:
It might be helpful to added a commented out location that the latitude and longitude represents, e.g. // Portland, OR. Totally optional, but would make it easier for others to immediately know where the map is centered.

Reply:
We were curious as well, as this was the center provided in the documentation for google-map-react. It's in the middle of the Neva river in St. Petersburg, Russia. 🤷‍♀️

Reply:
Might be fun for the team to decide on an interesting spot to use as the default location. There’s a story behind the default center I used for whats-near.me....